bluebottle/time_based/tasks.py

```diff
@@ -14,7 +14,7 @@
 
 
 @periodic_task(
-    run_every=(crontab(minute='*/15')),
+    run_every=(crontab(minute='*')),
```

Comment:
Is this for testing, or do we think it's not heavy to run it this often?

Reply:
That was for testing. I'll revert that.